packages/query/src/graphql/graphql.module.ts

```diff
@@ -184,7 +185,9 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
     }
 
     app.use(ExpressPinoLogger(PinoConfig));
-
+    if (argv['query-batch-limit'] !== undefined) {
```

Comment:
I think there needs to be more validation around this. What if values are negative or 0. 

I think this check can be moved into the middleware too.